target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -323,6 +323,9 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:3" "3:lan:2" "4:lan:1" "5:lan:4"
 		;;
+	tplink,tl-wpa8730-v1)
+		ucidef_set_interface_lan "eth0" "dhcp"
```

Comment:
I personally would prefer to remove the "dhcp" here, so the default config is more like what you would expect from OpenWrt.

Reply:
In my oppinion it's a device meant to be added to an existing network and should default to DHCP. Upstream also defaults to DHCP.